lib/components/forms/ClusterBuildForm.js

```diff
@@ -96,7 +103,7 @@ class ClusterBuildForm extends React.Component {
           spec: gkeSpec
         })
         const k8sSpec = {
-          domain: values.domain,
+          domain: showKubernetesOptions ? values.domain : '0.0.0.0',
```

Comment:
Am I correct is saying your gating on the `showKubernetesOptions` and defaulting to `0.0.0.0` is not enabled? ... I assuming that's to get around the field being required? ... Might be cleaner to drop the field as a required field for now in the crd

Reply:
Correct. Amending as discussed/